stacks/spica/src/function/interface.ts

```diff
@@ -69,5 +70,5 @@ export interface Trigger<T = any> {
 }
 
 export function emptySubscription(): Subscription {
-  return {trigger: {type: undefined, options: {}}, url: undefined};
+  return {trigger: {type: undefined, status: true, options: {}}, url: undefined};
```

Comment:
Do we need status for a subscription ? If we don't please make status property optional or create another interface for it.